tests/test_cbox.py

```diff
@@ -73,6 +74,55 @@ def hello(name):
     assert msg == ['test']
 
 
+def test_cmd_with_default_true_bool_argument():
+    msg = []
```

Comment:
add a test case for required bool argument case with 3 options:
1. invalid value
2. Truthy value
3. Falsy value